clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AsgReferenceCopier.groovy

```diff
@@ -59,7 +64,13 @@ class AsgReferenceCopier {
       if (startTime?.time > System.currentTimeMillis()) {
         request.withStartTime(startTime)
       }
-      targetAutoScaling.putScheduledUpdateGroupAction(request)
+
+      try {
+        targetAutoScaling.putScheduledUpdateGroupAction(request)
+      } catch (AlreadyExistsException e) {
+        // This should never happen as the name is generated with a UUID.
+        log.info("Scheduled action already exists on ASG, continuing: $request")
```

Comment:
Maybe a warn?

For better or worse I try and follow a logging convention like `"Scheduled action already exists on ASG (request: {})", request)`

🤷‍♂️